src/messages/MMgrBeacon.h

```diff
@@ -147,7 +147,15 @@ class MMgrBeacon final : public PaxosServiceMessage {
 
     encode(modules, payload);
     encode(mgr_features, payload);
-    encode(clients, payload, features);
+
+    std::vector<std::string> clients_names;
+    std::vector<entity_addrvec_t> clients_addrs;
+    for (const auto& i : clients) {
+      clients_names.push_back(i.first);
+      clients_addrs.push_back(i.second);
+    }
+    encode(clients_addrs, payload, features);
+    encode(clients_names, payload, features);
```

Comment:
This needs a comment to explain why the multimap encode/decode is not used. Namely: the addr vector needs to be first in order to produce a backwards compatible message (for older monitors).

Reply:
Done.